save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/utils/AvatarType.kt

```diff
@@ -7,6 +7,11 @@ import kotlinx.serialization.Serializable
  */
 @Serializable
 enum class AvatarType {
+    /**
+     * default
+     */
+    NONE,
```

Comment:
Please document what `NONE` actually means and what's its intended usage.

Reply:
Deleted this type